lib/wasix/src/bin_factory/binary_package.rs

```diff
@@ -69,7 +70,6 @@ pub struct BinaryPackage {
     pub webc_fs: Arc<dyn FileSystem + Send + Sync>,
     pub commands: Vec<BinaryPackageCommand>,
     pub uses: Vec<String>,
```

Comment:
Should this be using `Vec<PackageId>` eventually? We may need to create a ticket about this so we don't forget

Reply:
@theduke

Reply:
followup issue: https://github.com/wasmerio/wasmer/issues/4597